components/doc/carousel/index.js

```diff
@@ -549,6 +549,24 @@ const responsiveOptions = [
                                     <td>false</td>
                                     <td>Defines if scrolling would be infinite.</td>
                                 </tr>
+                                <tr>
+                                    <td>showIndicators</td>
```

Comment:
@ulasturann it looks like you are committing everything to your master branch instead of creating PR branches so all of your PR's are now showing the same changes